src/relay/op/memory/device_copy.cc

```diff
@@ -103,6 +107,20 @@ DeviceCopyProps GetDeviceCopyProps(const CallNode* call_node) {
     } else {
       return {call_node->args[0], src_dev_type, dst_dev_type};
     }
+  } else if (call_node->op == CallLoweredOp()) {
```

Comment:
I'm a bit nervous about this one since there's a lot of usage patterns such as

props = GetProps
if (prop.body.defined()) { .... MakeDeviceCopy(...) ... }

which would silently demote call_lowered copies to regular copies.

I think safer would be to have a GetLoweredDeviceCopyProps so it's always clear which form is being matched against.

Reply:
Talked with mark and we determined this is fine, he was just being a bit skittish

Reply:
I think that this chunk of code is not necessary since I also deal with CallLoweredOp in GetPrimitiveDeviceCopyProps, I'm not sure why I put it in. Removing it